packages/manager/modules/dbaas-logs/src/logs/detail/streams/home/logs-streams-home.controller.js

```diff
@@ -199,6 +199,20 @@ export default class LogsStreamsHomeCtrl {
     });
   }
 
+  /**
+   * navigates to the subscriptions page
+   *
+   * @param {object} stream, stream for which subscriptions should be managed
+   * @memberof LogsStreamsHomeCtrl
+   */
+  gotoSubscriptions(stream) {
```

Comment:
```suggestion
  goToSubscriptions(stream) {
```

Reply:
Done, also applied to `goToArchives` in the same file